man/systemctl.xml

```diff
@@ -550,6 +550,19 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><command>bind</command> <replaceable>UNIT</replaceable> <replaceable>PATH</replaceable> [<replaceable>PATH</replaceable>]</term>
+
+          <listitem><para>Bind mounts a file or directory from the host into the specified unit's view. The first path
+          argument is the source file or directory on the host, the second path argument is the destination file or
+          directory in the unit's view. When the latter is omitted, the destination path in the unit's view is the same as
+          the source path on the host. When combined with the <option>--read-only</option> switch, a ready-only bind
+          mount is created. When combined with the <option>--mkdir</option> switch, the destination path is first created
+          before the mount is applied. Note that this option is currently only supported for units that run within a mount
+          namespace (eg: with <option>RootImage=</option>). This command supports bind mounting directories, regular files,
```

Comment:
maybe mention `<option>PrivateMounts=</option>` explicitly too. given it's the least involved, and most explicit ways to request a private mount namespace for a service.

maybe eg → e.g.?

Reply:
sounds good, both fixed